instill/pipeline/v1alpha/pipeline.proto

```diff
@@ -173,13 +124,13 @@ message ListPipelineRequest {
   optional int32 page_size = 1 [ (google.api.field_behavior) = OPTIONAL ];
   // Page token
   optional string page_token = 2 [ (google.api.field_behavior) = OPTIONAL ];
-  // PipelineView view (default is PIPELINE_VIEW_BASIC)
-  optional PipelineView view = 3 [ (google.api.field_behavior) = OPTIONAL ];
+  // View view (default is PIPELINE_VIEW_BASIC)
+  optional View view = 3 [ (google.api.field_behavior) = OPTIONAL ];
 }
 
 // ListPipelineResponse represents a response for a list of pipelines
 message ListPipelineResponse {
-  // A list of pipeline instances
+  // A list of pipeline resources
   repeated Pipeline pipelines = 1;
   // Next page token
   string next_page_token = 2;
```

Comment:
```suggestion
  string next_page_token = 2;
  // Total count of pipeline resources
  int32 total_size = 3;
```

Reply:
See [here](https://google.aip.dev/132#response-message).
> The message may include a int32 total_size (or int64 total_size) field with the number of items in the collection.
The value may be an estimate (the field should clearly document this if so).
If filtering is used, the total_size field should reflect the size of the collection after the filter is applied.


This will help users know the number of pipelines easily.